packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -90,7 +90,7 @@ export class Eth2Gossipsub extends GossipSub {
     // https://github.com/ethereum/consensus-specs/blob/v1.1.10/specs/phase0/p2p-interface.md#the-gossip-domain-gossipsub
     super(modules.libp2p.services.components, {
       globalSignaturePolicy: SignaturePolicy.StrictNoSign,
-      allowPublishToZeroPeers: allowPublishToZeroPeers,
+      allowPublishToZeroTopicPeers: allowPublishToZeroPeers,
```

Comment:
This is great change to get in, previous error message caused so much user confusion

Reply:
Note: we have a cli flag `--allowPublishToZeroPeers`, could alias it but since it mostly used for 0 total peers setups shouldn't matter